repo/packages/K/kafka-manager/0/config.json

```diff
@@ -0,0 +1,57 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "type": "object",
+  "properties": {
+    "service":{
+        "type":"object",
+        "description": "DC/OS service configuration properties",
+        "properties":{
+            "name" : {
+                "description":"The name of the kafka-manager service instance.",
+                "type":"string",
+                "default":"kafka-manager"
+            }
+        }
+    },
+    "kafka-manager": {
+      "additionalProperties": false,
+      "description": "Kafka-manager configuration properties",
+      "type": "object",
+      "properties": {
+        "cpus": {
+            "description": "CPU shares to allocate",
+            "type": "number",
+            "default": 1.0,
+            "minimum": 0.1
+        },
+        "mem": {
+            "description": "Memory to allocate",
+            "type": "number",
+            "default": 1024.0,
+            "minimum": 1024.0
+        },
+        "zk": {
+            "type": "string",
+            "description": "Zookeeper URL for Kafka-manager",
+            "default": "master.mesos:2181"
```

Comment:
We've got a new, better zk url:

    zk-1.zk:2181,zk-2.zk:2181,zk-3.zk:2181,zk-4.zk:2181,zk-5.zk:2181